specs/centos_ci.adoc

```diff
@@ -2,28 +2,49 @@
 
 = CentOS CI
 
-CentOS CI should be configured to launch applicable set of tests as an
-acceptance tests for every new built rpm package.
+We need to make sure that new code doesn't break existing codebase and check
+that new features works as early as possible. This can be done by running
+acceptance integration tests automatically, using CentOS CI infrastructure.
 
 == Problem description
 
-None
+CentOS CI
+will be configured to launch acceptance integration test run for every new
+nightly rpm build from tendrl nightly copr project. As set of implemented
+features in tendrl codebase grows, the set of acceptance tests is extended to
+cover new features.
 
 == Use Cases
 
-* Acceptance testing for every new rpm package.
+* Acceptance testing for every new build from tendrl copr nightly project.
+* The same set of tests should be run for stable builds from current tendrl
+  copr as well.
 
 == Proposed change
 
-* CentOS CI host provisioning system have to be utilized for testing machines
+* New copr tendrl project with nightly builds needs to be created. Right now
```

Comment:
No particular reason for copr, we could use centos koji instance as well. This was just my initial proposal.